### ISSUE TITLE
Is it possible to generate a pytest HTML report on a timeout?

### ISSUE THREAD

**Issue by mhoffman2016** [pytest-dev/pytest-html#184]
I want to combine pytest-html and pytest-timeout. I'm operating on Windows, so I have to use the thread-method of timing out from pytest-timeout, which causes a full-stop to all testing. I want an html from pytest-html to be generated in this instance, but was unsure of the plausibility of this from hierarchy issues with the interrupt.

**Comment by davehunt** [comment]
I don't think this is currently possible, but I'm not familiar enough with pytest-timeout. I'm also going to be less active on this project due to a change in my role. Hopefully somebody may be able to help you.
